docs/criteria/accelerators/beta.md

```diff
@@ -1,5 +1,22 @@
 # Spice.ai OSS Data Accelerators - Beta Release Criteria
 
+## Contents
+
+- [Description](#description)
+- [Beta Quality Accelerators](#beta-quality-accelerators)
+- [Beta Release Criteria](#beta-release-criteria)
+  - [Bug Levels](#bug-levels)
+    - [Major Bug](#major-bug)
+    - [Minor Bug](#minor-bug)
+  - [Feature Complete](#feature-complete)
+  - [Test Coverage](#test-coverage)
+    - [General](#general)
+    - [TPCH](#tpch)
```

Comment:
Let's be consistent with the naming, `TPC-H`, etc. to make below under test coverage.